### ISSUE TITLE
Update Examples from 0.9.x to 1.0

### ISSUE THREAD

**Issue by lusloher** [jina-ai/jina#2089]
Test and update all examples to fit the latest release

TODO:
- Fix the jina version in `requirements.txt` to `jina[scipy, http]==1.0.x`
- Fix the jina tag in `Dockerfile` to `FROM jinaai/jina:1.0.x`
- Update Flow YAML config files to 1.0. Reference: https://docs.jina.ai/chapters/yaml/flow/
- Build and push the Docker image 

Checklist:
- [x] advanced-vector-search https://github.com/jina-ai/examples/pull/419
- [x] audio-search  https://github.com/jina-ai/examples/pull/416			
- [x] cross-modal-search @bwanglzu https://github.com/jina-ai/examples/pull/409
- [x] fashion-example-query. https://github.com/jina-ai/examples/pull/417
- [x] multimodal-search-tirg https://github.com/jina-ai/examples/pull/420
- [x] multires-lyrics-search
- [x] object-search	https://github.com/jina-ai/examples/pull/407
- [x] pokedex-with-bit https://github.com/jina-ai/examples/pull/418
- [ ] 🚧 tumblr-gif-search  https://github.com/jina-ai/examples/pull/415 (blocked by waiting for jina `v1.0.9`)
- [x] wikipedia-sentences
- [x] wikipedia-sentences-incremental

**Comment by lusloher** [comment]
Link to feature: https://jinaai.productboard.com/feature-board/planning/features/7141143

**Comment by FionnD** [comment]
Hey @nan-wang 
I see you were assigned this on GH, are you taking this task or should someone else?

**Comment by bwanglzu** [comment]
please ignore cross model search example &  multimodal search example, I'm working on it.

**Comment by bwanglzu** [comment]
[PR](https://github.com/jina-ai/examples/pull/409) for cross model search example (in progress):

**Comment by Yongxuanzhang** [comment]
https://github.com/jina-ai/examples/pull/426
new update for object search

**Comment by FionnD** [comment]
I am closing this issue since it's no longer relevant. We are now working towards fixing 1.10
